bin/sugar-launch

```diff
@@ -93,4 +93,4 @@ if args.debug:
     cmd_args.extend(act_args)
 
 os.chdir(str(activity.get_path()))
-os.execvpe(cmd_args[0], cmd_args, activityfactory.get_environment(activity))
+os.execvpe(cmd_args[0], cmd_args, activityfactory.get_environment(activity))
```

Comment:
Change removed newline at end of file.  Did you review your change before commit?